internal/ingress/controller/controller.go

```diff
@@ -164,6 +164,11 @@ func (n *NGINXController) syncIngress(item interface{}) error {
 		PassthroughBackends: passUpstreams,
 	}
 
+	if n.isForceNoReload() {
+		glog.V(3).Infof("force no reload, skipping backend reload")
+		return nil
+	}
```

Comment:
this is probs fine for hacking but I think we should make sure the configuration on disk gets updated.

iirc it is done in OnUpdate function

we should also make sure n.runningConfig is updated